cluster/gce/gci/configure-helper.sh

```diff
@@ -1644,6 +1644,9 @@ function start-node-problem-detector {
     local -r sm_config="${KUBE_HOME}/node-problem-detector/config/systemd-monitor.json"
     local -r ssm_config="${KUBE_HOME}/node-problem-detector/config/system-stats-monitor.json"
 
+    # replace the known-modules.json file path in system-stats-monitor.json with the absolute path
```

Comment:
I'm OK with this fix as long as we put a TODO here referencing this PR: https://github.com/kubernetes/kubernetes/pull/101629 and the next version of NPD where this `sed` hack needs to be removed.